src/main/kotlin/org/opensearch/reportsscheduler/ReportsSchedulerPlugin.kt

```diff
@@ -77,6 +79,12 @@ class ReportsSchedulerPlugin : Plugin(), ActionPlugin, JobSchedulerExtension {
         return settingList
     }
 
+    override fun getSystemIndexDescriptors(settings: Settings): Collection<SystemIndexDescriptor> {
+        return listOf(
+            SystemIndexDescriptor(".opendistro-reports-*", "Reports Scheduler Plugin system index pattern")
```

Comment:
Should we explicitly list these out? https://github.com/search?q=repo%3Aopensearch-project%2Freporting%20.opendistro-reports&type=code

Reply:
IMO yes. Plugins should only register a pattern as a system index if they create a variable amount of system indices based on that pattern.

The best example of that is multi-tenancy with kibana where there is an index per tenant. i.e. `.kibana-<tenant>`

Reply:
Should I update this?

Reply:
@cwperks Yes if you could. Since this is a new way of doing things lets follow the best practices.

Reply:
@derek-ho updated